product-create-search/src/BusinessLogic/SendToKafka.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+
+namespace App\BusinessLogic;
+
+
+use Kafka\Producer;
+use Kafka\ProducerConfig;
+use Symfony\Component\Serializer\SerializerInterface;
+
+class SendToKafka implements SendToKafkaI
+{
+    private $crudMysql;
+    private $status;
+    private $serializer;
+
+    public function __construct(CrudMysqLI $crudMysqli,  SerializerInterface $serializer)
+    {
+        $this->crudMysql = $crudMysqli;
+        $this->serializer = $serializer;
+    }
+
+    public function sendToKafka($status)
+    {
+        $this->status = $status;
+
+        $config = ProducerConfig::getInstance();
```

Comment:
configuration should be in configuration file, remember that we will have many environments  :) 
so in general we will have files like
app.config.dest => this will be add to git
app.config     => will be configured on each machine , will not be added to git